torch/testing/_internal/common_methods_invocations.py

```diff
@@ -281,6 +281,15 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad):
                                     low=None, high=None,
                                     requires_grad=False))),)
 
+def np_sinc_with_fp16_as_fp32(x):
+    # Wraps numpy's sinc function so that fp16 values are promoted to fp32
+    # before sinc is invoked. Context: numpy's sinc returns NaN when evaluated
+    # at 0 for fp16.
+    if x.dtype in [np.float16]:
```

Comment:
-> `if x.dtype is np.float16:`

Reply:
Testing with `is` fails because of a bug https://stackoverflow.com/questions/26921836/correct-way-to-test-for-numpy-dtype. Using `==` here instead.